sadcowpng/sharedspaces/static/account.css

```diff
@@ -14,12 +14,13 @@
 
     body {
         background-color: #D2FDFF;
+        overflow: auto;
     }
 
     .container-fluid{
         background-color: #303C6C;
         width: 1000px;
```

Comment:
might want to make width auto as well fixes the issue with navbar size

Reply:
when i get to mobile/desktop story i will auto everything